webpack/scenes/ContentViews/Details/ContentViewDetailActions.js

```diff
@@ -169,6 +170,18 @@ export const getFilesComparison = (versionOne, versionTwo, params) => {
   });
 };
 
+export const getContentComparison = (pluralLabel, versionOne, versionTwo, params) => {
```

Comment:
```suggestion
export const getGenericContentComparison = (pluralLabel, versionOne, versionTwo, params) => {
```

Reply:
@tazhibaevaaliya : Just add this to your other PR and we can merge this so you don't can stop worrying about rebasing and stuff..